### PR TITLE
cli: Add thin arbiter support

### LINKED ISSUE #701: Add CLI support for thin arbiter

### PR DESCRIPTION
**Pull request by prashanthpai**
Closes #701

Signed-off-by: Prashanth Pai <ppai@redhat.com>

### CODE REVIEW

File: glustercli/cmd/volume-create.go [hunk 1]
```diff
@@ -110,3 +128,19 @@ func volumeCreateCmdRun(cmd *cobra.Command, args []string) {
...
+
+	s := strings.Split(thinArbiter, ":")
+	if len(s) != 2 && len(s) != 3 {
+		return fmt.Errorf("Thin arbiter brick must be of the form <host>:<brick> or <host>:<brick>:<port>")
+	}
+
+	// TODO: Handle this in a generic way, just like other volume set
+	// options that we're gonna allow later.
+	req.Options = map[string]string{
+		"afr.thin-arbiter": thinArbiter,
```
Comment (aravindavk) [inline glustercli/cmd/volume-create.go]: Use "replicate.thin-arbiter" so that Volgen automatically adds to Volfile

Comment (aravindavk) [reply glustercli/cmd/volume-create.go]: Looks like these options may become issue with afr prefix. None of the options added with afr prefix will appear in generated volfile under `type cluster/replicate`

Comment (prashanthpai) [reply glustercli/cmd/volume-create.go]: Okay. "Replicate" will work as the afr xlator is loaded under both (afr and replicate) names: https://github.com/gluster/glusterd2/blob/master/glusterd2/xlator/load.go#L169

Comment (aravindavk) [reply glustercli/cmd/volume-create.go]: Not related to this patch, but other replicate options set accepts both `afr.*` and `replicate.*`, any suggestion to normalize these during Volume set?

Comment (prashanthpai) [reply glustercli/cmd/volume-create.go]: We can take a call on that later may be. `replicate` sounds more understandable though than `afr`. The same applies to `dht` vs `distribute`.

**Pull request merged**